### DISCUSSION TITLE
Compatible with ESP32-PICO-D4?

### DISCUSSION THREAD

**Discussion by ghost** [Bodmer/TFT_eSPI#1889] [Compatible displays and setup files]
I am currently working on a very small form factor audio-reactive LCD device and found the M5 STAMP (ESP32-PICO-D4) to be a suitable choice for this project. After trying a few combinations of modifying the setup header on this library to no avail, I am wondering if any drivers for an ST7789 based are compatible with this device. I am relatively new to ESP32, coming from more basic hobbyist Arduino platforms and any advice is appreciated!

**Comment by Bodmer** [comment]
The D4 should work OK with the library as it is just the original ESP32 with FLASH integrated in the chip.
